File: loans/tests/pages/add_business.py
from selenium.webdriver.support.select import Select

class AddBusinessPage():

    @staticmethod
    def complete_form(driver, crn, business_name, sector, address_1, address_2, city, postcode):
        driver.find_element_by_id('input_crn').send_keys(crn)
        driver.find_element_by_id('input_name').send_keys(business_name)
        sector_select = Select(driver.find_element_by_id('input_sector'))
        sector_select.select_by_visible_text(sector)
        driver.find_element_by_id('input_address_1').send_keys(address_1)
        driver.find_element_by_id('input_address_2').send_keys(address_2)
        driver.find_element_by_id('input_city').send_keys(city)
        driver.find_element_by_id('input_postcode').send_keys(postcode)
        driver.find_element_by_id('submit').click()

File: loans/static/loans/controllers/add_business.js

$(document).ready(function() {

  $("#submit").click(function() {

    $.ajax({
      url: "api/v1/register",
      type: "POST",
      contentType: "application/json",
      data: JSON.stringify(JSON.stringify({
        'crn': $("#input_crn").val(),
        'business_name': $('#input_name').val(),
        'sector': $('#input_sector').val(),
        'address_1': $('#input_address_1').val(),
        'address_2': $('#input_address_2').val(),
        'city': $('#input_city').val(),
        'postcode': $('#input_postcode').val(),
      })),
      statusCode: {
        200: function(data) {
          document.location.replace("apply/loan-application/3/" + $("#input_crn").val());
        },
        400: function(data) {
          $("#error-message").html("ERROR: " + data.responseText);
          $("#error-message").css("display", "");
        }
      }
    }); // $.ajax

  }); // #submit.click()

});
